src/Sentry.Unity/WebGL/SentryWebGL.cs

```diff
@@ -0,0 +1,227 @@
+using System;
+using System.Collections;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Sentry.Extensibility;
+using Sentry.Http;
+using Sentry.Infrastructure;
+using Sentry.Internal;
+using Sentry.Internal.Http;
+using Sentry.Protocol.Envelopes;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace Sentry.Unity.WebGL
+{
+    /// <summary>
+    /// Configure Sentry for WebGL
+    /// </summary>
+    public static class SentryWebGL
+    {
+        /// <summary>
+        /// Configures the WebGL support.
+        /// </summary>
+        /// <param name="options">The Sentry Unity options to use.</param>
+        public static void Configure(SentryUnityOptions options)
+        {
+            options.DiagnosticLogger?.LogDebug("Updating configuration for Unity WebGL.");
+
+            // Caching transport relies on a background thread
+            options.CacheDirectoryPath = null;
+            // Note: we need to use a custom background worker which actually doesn't work in the background
+            // because Unity doesn't support async (multithreading) yet. This may change in the future so let's watch
+            // https://docs.unity3d.com/2019.4/Documentation/ScriptReference/PlayerSettings.WebGL-threadsSupport.html
+            options.BackgroundWorker = new WebBackgroundWorker(options, SentryMonoBehaviour.Instance);
+
+            options.CrashedLastRun = () => false; // no way to recognize crashes in WebGL yet
```

Comment:
Note: Align the concept of 'crash' sessions with the JS SDK but it's never about 'on-restart' which is what this field is used for

```suggestion
            options.CrashedLastRun = () => false; // Since there's no offline caching, this won't be called anyway
```

Reply:
The suggested comment is not true anymore because it turns out we can have offline caching just fine (uses indexeddb automatically as a filesystem)
<img width="1502" alt="image" src="https://user-images.githubusercontent.com/6349682/160633137-8c335cf2-958f-4f01-ab83-31a27a7f3d02.png">